RasimKhusaenov/my_own_blog
# Add company's registration page

### Summary

- Added company's registration page
- Added `friendly-id` gem
- Added `slug` attribute to Company
- Company's url have `/companies/company-official-name` format now

### How it works

![image](https://user-images.githubusercontent.com/35406008/173195329-94a0005c-ab5b-42e3-b27c-92bf4797bb4c.png)
![image](https://user-images.githubusercontent.com/35406008/173195338-c5b8365f-5551-4916-b1df-6c4cc52400de.png)

### Test plan

List of steps to manually test introduced functionality:

- Go to https://my-own-blog-pr-57.herokuapp.com/companies/microsoft/registrations/new
- Fill form
- Click "Sign Up"
- Make sure that user have been created and signed in

### Deploy notes

- Run `bin/rails migrate`
- Run `Company.find_each(&:save)`

### Checklist:


### References
- Resolves #36 Issue
